packages/app-spec-squad-chess/src/index.js

```diff
@@ -16,7 +16,8 @@ async function init() {
   const formatDefs = await metastore.getGameFormats(settings.gameAddress)
   state.formats = formatDefs.map(def => def.Format)
   const format = state.formats[0] // TODO build in a format selection interface
-  console.log(format)
+  chess.registerFormat(JSON.parse(format.data))
```

Comment:
Let's make `registerFormat` take a format data signature with actual piece definitions in it instead of addresses, and then deprecate the `registerPieces` function.